lab-gurpreet/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "cowsay-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
You forgot to include gulp as a devDependency; when I try to run gulp I get an error.